views/admin/exhibits/exhibit-metadata-form.php

```diff
@@ -63,6 +63,17 @@
                 <?php echo $this->formCheckbox('use_summary_page', $exhibit->use_summary_page, array(), array('1', '0')); ?>
             </div>
         </div>
+        <div id="cover-image-container" class="field">
+            <div class="two columns alpha">
+                <?php echo $this->formLabel('cover_image', __('Cover Image')); ?>
+            </div>
+            <div class="five columns omega inputs">
+                <p class="explanation">
+                  <?php echo __("Choose an item to represent this exhibit. Shown on the Browse Exhibits page and on the home page when the exhibit is featured."); ?>
+                </p>
+                <?php echo $this->partial('files/cover-image.php', array('file' => $exhibit->getFile())); ?>
```

Comment:
Sounds like a good idea to me.  I have added a `getCoverImage()` accessor to the Exhibit model and it's being used on the form now.